script/periphery/CopyOpenSemibooks.s.sol

```diff
@@ -4,6 +4,7 @@ pragma solidity ^0.8.13;
 import "mgv_src/periphery/MgvReader.sol";
 import {Deployer} from "mgv_script/lib/Deployer.sol";
 import {MgvStructs, OLKey} from "mgv_src/MgvLib.sol";
+import "mgv_lib/Debug.sol";
```

Comment:
Should this be deleted?

```suggestion
```

Reply:
No - they implicitly got console from reader, and I removed the import in reader, making these scripts fail. We could have imported only console, but the debug toString stuff is nice to have.